hockeysdk/src/main/java/net/hockeyapp/android/objects/CrashDetails.java

```diff
@@ -56,6 +56,10 @@
 
     public CrashDetails(String crashIdentifier) {
         this.crashIdentifier = crashIdentifier;
+        appStartDate = new Date();
```

Comment:
@ranterle - I hear what you are saying.  How about the 2 lines below?
```
isXamarinException = false;
throwableStackTrace = "";
```

If I gave you a new PR with just those 2 lines would you merge it?  If not, how about just `isXamarinException = false;` ?  The other constructors all set a value for `isXamarinException`.